src/main/scala/lms/transformation/tensor2.scala

```diff
@@ -0,0 +1,115 @@
+package lms
```

Comment:
why this file is called `tensor2.scala`? is there a `tensor1.scala` or something similar?

Reply:
there was but it was not committed.
 I can rename this :)